src/components/KymaCompanion/components/KymaCompanion.tsx

```diff
@@ -1,18 +1,52 @@
 import { useTranslation } from 'react-i18next';
 import { Button, Card, Title } from '@ui5/webcomponents-react';
-import { useRecoilState } from 'recoil';
+import { useRecoilState, useRecoilValue, useSetRecoilState } from 'recoil';
 import {
   ShowKymaCompanion,
   showKymaCompanionState,
 } from 'components/KymaCompanion/state/showKymaCompanionAtom';
 import Chat from './Chat/Chat';
+import { useEffect, useState } from 'react';
+import { sessionIDState } from '../state/sessionIDAtom';
```

Comment:
Could it be like this?
```suggestion
import { sessionIDState } from 'state/sessionIDAtom';
```

Reply:
we can move the state file if you want, so instead of having it as a subdirectory of the Companion component
`src/components/KymaCompanion/state/sessionIDAtom.ts` we can move it to the global state directory
`src/state/companion/sessionIDAtom.ts`

Reply:
I don't have strong opinion, but probably this could make sense, and will be easier to maintain, to have all of the global states in one place.